ambry-replication/src/main/java/com/github/ambry/replication/ReplicaThread.java

```diff
@@ -494,19 +514,25 @@ public void replicate() {
                 // Must have just been stopped, just skip it and move on.
                 logger.info("Local store not started for remote replica: {}", remoteReplicaInfo.getReplicaId());
                 exchangeMetadataResponseList.add(new ExchangeMetadataResponse(ServerErrorCode.Temporarily_Disabled));
+                remoteReplicaInfo.setExchangeMetadataResponse(
```

Comment:
One minor comment that should be considered in future PR: if it metadata exchange fails (i.e. due to replica unavailable) within current replication cycle, we may allow standby to send metadata request again in next cycles.